store/storetest/thread_store.go

```diff
@@ -54,26 +54,26 @@ func testThreadStorePopulation(t *testing.T, ss store.Store) {
 		o := model.Post{}
 		o.ChannelId = c.Id
 		o.UserId = u.Id
-		o.Message = "zz" + model.NewId() + "b"
+		o.Message = NewTestId() + "b"
```

Comment:
Please remove the `b` as well here and everywhere else.

Reply:
Can you please guide me how to go about it?
Should I create a new pull request?

Reply:
Or should I commit the new changes to this pull request itself?

Reply:
You can commit your new changes to this PR itself. Create a new commit and push it. The PR will update itself.

Reply:
Ok thank you.